# Dennis Tobar <dennis.tobar@gmail.com>, 2015. #zanata
# Eduardo Mayorga Téllez <e@mayorgalinux.com>, 2016. #zanata
# Máximo Castañeda Riloba <mcrcctm@gmail.com>, 2016. #zanata
# Omar Berroterán S. <omarberroteranlkf@gmail.com>, 2016. #zanata
# William  Moreno Reyes <williamjmorenor@gmail.com>, 2016. #zanata
# Máximo Castañeda Riloba <mcrcctm@gmail.com>, 2017. #zanata
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2018-04-06 07:47+0200\n"
"PO-Revision-Date: 2017-01-30 02:57-0500\n"
"Last-Translator: Máximo Castañeda Riloba <mcrcctm@gmail.com>\n"
"Language-Team: Spanish\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: es\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"X-Generator: Zanata 3.9.6\n"

#. Translators: This string is only used in unit tests.
#: ../plugins/system_upgrade.py:38
msgid "the color of the sky"
msgstr "el color del cielo"

#: ../plugins/system_upgrade.py:54
msgid "Need a --releasever greater than the current system version."
msgstr "Necesita una --releasever mayor que la versión actual del sistema."

#. Translators: do not change "reboot" here
#: ../plugins/system_upgrade.py:56
msgid ""
"Download complete! Use 'dnf system-upgrade reboot' to start the upgrade.\n"
"To remove cached metadata and transaction use 'dnf system-upgrade clean'"
msgstr ""

#: ../plugins/system_upgrade.py:59
msgid "Sorry, you need to use 'download --releasever' instead of '--network'"
msgstr "Use 'download --releasever' en lugar de '--network'"

#: ../plugins/system_upgrade.py:235
msgid "The following boots appear to contain upgrade logs:"
msgstr ""
"Estos arranques parecen contener registros de actualizaciones de sistema:"

#: ../plugins/system_upgrade.py:245
msgid "-- no logs were found --"
msgstr "-- no se encontró ningún registro de actualizaciones --"

#: ../plugins/system_upgrade.py:260
msgid "Cannot find logs with this index."
msgstr "No se encontraron registros con ese índice."

#: ../plugins/system_upgrade.py:269
msgid "Unable to match systemd journal entry"
msgstr ""

#: ../plugins/system_upgrade.py:288
msgid "Prepare system for upgrade to a new release"
msgstr "Preparar el sistema para una nueva versión"

#: ../plugins/system_upgrade.py:298
msgid "keep installed packages if the new release's version is older"
msgstr ""
"mantener los paquetes instalados si la versión del nuevo sistema es anterior"

#: ../plugins/system_upgrade.py:302
msgid "which logs to show"
msgstr ""

#: ../plugins/system_upgrade.py:345
msgid ""
"Before you continue ensure that your system is fully upgraded by running "
"\"dnf --refresh upgrade\". Do you want to continue"
msgstr ""

#: ../plugins/system_upgrade.py:349
msgid "Operation aborted."
msgstr ""

#: ../plugins/system_upgrade.py:400
msgid "system is not ready for upgrade"
msgstr "el sistema no está listo para la actualización"

#: ../plugins/system_upgrade.py:402
msgid "upgrade is already scheduled"
msgstr "la actualización ya está planificada"

#: ../plugins/system_upgrade.py:407
msgid "trigger file does not exist. exiting quietly."
msgstr ""

#: ../plugins/system_upgrade.py:410
msgid "another upgrade tool is running. exiting quietly."
msgstr ""
"existe otra herramienta de actualización en ejecución. Saliendo "
"silenciosamente."

#. Translators: do not change "reboot" here
#: ../plugins/system_upgrade.py:416
msgid "use 'dnf system-upgrade reboot' to begin the upgrade"
msgstr "use 'dnf system-upgrade reboot' para iniciar la actualización"

#: ../plugins/system_upgrade.py:436
msgid "Rebooting to perform upgrade."
msgstr "Reiniciando para proceder a la actualización."

#: ../plugins/system_upgrade.py:457 ../plugins/system_upgrade.py:463
msgid "Starting system upgrade. This will take a while."
msgstr "Iniciando la actualización del sistema. Llevará un rato."

#: ../plugins/system_upgrade.py:489
#, python-format
msgid "Unable to match package: %s"
msgstr ""

#: ../plugins/system_upgrade.py:494
msgid "Unable to match some of packages"
msgstr ""

#: ../plugins/system_upgrade.py:497
msgid "Cleaning up downloaded data..."
msgstr "Limpiando los datos descargados..."

#: ../plugins/system_upgrade.py:531
msgid "Download finished."
msgstr "Terminó la descarga."

#: ../plugins/system_upgrade.py:535 ../plugins/system_upgrade.py:536
msgid "Upgrade complete! Cleaning up and rebooting..."
msgstr "¡Actualización completa! Limpiando y reiniciando..."

#: ../plugins/tracer.py:70
msgid "For more information run:"
msgstr "Para mayor información ejecute:"

#: ../plugins/tracer.py:97
msgid "Call to Tracer API ended unexpectedly:"
msgstr "La llamada a la API tracer acabó con error:"

#: ../plugins/tracer.py:99
msgid ""
"Please visit https://github.com/FrostyX/tracer/issues and submit the issue. "
"Thank you"
msgstr ""
"Vaya a https://github.com/FrostyX/tracer/issues y envíe un informe de error."

#: ../plugins/tracer.py:101
msgid "We apologize for any inconvenience"
msgstr "Lamentamos cualquier incoveniente"

#: ../plugins/rpm_conf.py:69
msgid "Adding '{}' to list of handling packages for rpmconf"
msgstr "Agregando '{}'  a la lista de paquetes manejados por rpmconf"

#: ../plugins/rpm_conf.py:75
msgid "rpmconf plugin will not run in non-interactive mode"
msgstr "El complemento rpmconf sólo se ejecuta en modo interactivo"

#: ../plugins/rpm_conf.py:88
msgid "ignoring sys.exit from rpmconf due to missing MERGE variable"
msgstr "se descarta sys.exit de rpmconf al faltar la variable MERGE"

#: ../plugins/rpm_conf.py:92
msgid "ignoring sys.exit from rpmconf due to missing file"
msgstr "se descarta sys.exit de rpmconf por la falta de un archivo"

#: ../plugins/snapper.py:49
#, python-format
msgid "connect to snapperd failed: %s"
msgstr "conexión a snapperd fallida: %s"

#: ../plugins/snapper.py:55
msgid "creating pre_snapshot"
msgstr ""

#: ../plugins/snapper.py:61
#, python-format
msgid "created pre_snapshot %d"
msgstr ""

#: ../plugins/snapper.py:65
#, python-format
msgid "creating pre_snapshot failed: %s"
msgstr ""

#: ../plugins/snapper.py:71
msgid "skipping post_snapshot because creation of pre_snapshot failed"
msgstr ""

#: ../plugins/snapper.py:77
msgid "creating post_snapshot"
msgstr ""

#: ../plugins/snapper.py:82
#, python-format
msgid "created post_snapshot %d"
msgstr ""

#: ../plugins/snapper.py:86
#, python-format
msgid "creating post_snapshot failed: %s"
msgstr ""

#: ../plugins/kickstart.py:59
msgid "Install packages defined in a kickstart file on your system"
msgstr "Instalar los paquetes definidos en el archivo kickstart en su sistema"

#: ../plugins/kickstart.py:64
msgid "kickstart file"
msgstr "archivo kickstart"

#: ../plugins/kickstart.py:82
#, python-format
msgid "file cannot be parsed: %s"
msgstr "no se puede procesar el archivo: %s"

#: ../plugins/kickstart.py:99
#, python-format
msgid "No package %s available."
msgstr "No hay ningún paquete %s disponible."

#: ../plugins/kickstart.py:104
msgid "Nothing to do."
msgstr "Nada por hacer."

#: ../plugins/torproxy.py:32
msgid "Disabling torproxy plugin: cannot connect to the Tor network"
msgstr ""
"Desactivando el complemento torproxy: no se puede conectar con la red Tor"
